content/community/steering/index.md

```diff
@@ -1,32 +1,32 @@
 # Global Galaxy Steering Committee
 
-**Keywords:** Strategic emphasis, medium and long term planning, international opportunities
+**Keywords:** Strategic emphasis, medium and long term planning, international opportunities, community
 
 [Galaxy](/) is a mature, successful research software infrastructure project, with a large and diverse global audience. Changes in [project governance structures](/community/governance/) are needed to help adapt to increasing complexity.
 
-<div class="lead">
-
 The [Galaxy Executive Board](/community/exec/) has formed the Global Galaxy Steering Committee (GGSC) to represent the interests of communities engaged with the project, and to implement efficient consultative project governance structures for that purpose.
 
-</div>
+---
 
-<br />
+## News and Announcements
 
+**January 14, 2022**
+### Call for (self) nomination - Want to learn more about how *you* can contribute to the GGSC?
 
+The GGSC provides opportunities for community members willing to initiate and lead new community focussed initiatives.
 
-## Community consultation - help wanted!
+Community members who have ideas and strong feelings about one or more new projects that they would be willing to lead, or who would like to join existing
+working groups, please [contact the Chair](mailto:ross.lazarus@gmail.com) to discuss how the GGSC can help them achieve their goals.
```

Comment:
same comment as in the post: working groups or subcommittees?

Reply:
working groups - in the long run that's what makes more sense to me for integration to make it less confusing for newcomers.